docs/whats-new/dotnet-docs-2022-07-01.md

```diff
@@ -0,0 +1,114 @@
+- [dotnet dev-certs](../core/tools/dotnet-dev-certs.md)
+- [Changes to reflection invoke API exceptions](../core/compatibility/core-libraries/7.0/reflection-invoke-exceptions.md)
+
+## C# language
+
+### New articles
+
+- [required modifier (C# Reference)](../csharp/language-reference/keywords/required.md)
+- [Resolve nullable warnings](../csharp/language-reference/compiler-messages/nullable-warnings.md)
+
+### Updated articles
+
+- [Properties](../csharp/properties.md) - C# 11: required members
+
+## F# language
+
+### New articles
+
+- [FS0703: Expected type parameter, not unit-of-measure parameter](../fsharp/language-reference/compiler-messages/fs0703.md)
+
+## Microsoft Orleans
+
+### New articles
+
+- [Deploy Orleans to Azure Container Apps](../orleans/deployment/deploy-to-azure-container-apps.md)
+
+## Community contributors
+
+The following people contributed to the .NET docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).
+
+- [GitHubPang](https://github.com/GitHubPang) ![There were 10 pull requests merged by .](https://img.shields.io/badge/Merged%20Pull%20Requests-10-green)
+- [pkulikov](https://github.com/pkulikov) - Petr Kulikov ![There were 6 pull requests merged by Petr Kulikov.](https://img.shields.io/badge/Merged%20Pull%20Requests-6-green)
+- [petarmari266](https://github.com/petarmari266) ![There were 2 pull requests merged by .](https://img.shields.io/badge/Merged%20Pull%20Requests-2-green)
+- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor ![There were 2 pull requests merged by Youssef Victor.](https://img.shields.io/badge/Merged%20Pull%20Requests-2-green)
+- [abhith](https://github.com/abhith) - Abhith Rajan ![There were 1 pull requests merged by Abhith Rajan.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [berkid89](https://github.com/berkid89) - Daniel Berki ![There were 1 pull requests merged by Daniel Berki.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [cermakp](https://github.com/cermakp) - Pavel Čermák ![There were 1 pull requests merged by Pavel Čermák.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [chrisnas](https://github.com/chrisnas) ![There were 1 pull requests merged by .](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [eatdrinksleepcode](https://github.com/eatdrinksleepcode) - David Nelson ![There were 1 pull requests merged by David Nelson.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [FlyingDutchman1965](https://github.com/FlyingDutchman1965) ![There were 1 pull requests merged by .](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [github-actions](https://github.com/github-actions) -  ![There were 1 pull requests merged by .](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
```

Comment:
I'm torn as to whether or not this should be listed, I kind of lean towards _it should_. I mean, why not, someone wrote some sort of automation that caused this to run.